bundles/CoreBundle/translations/en.json

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
that file should not be there anymore ... was changed to yaml

Reply:
removed with 7e9420ca4607c93d98e20e2f3a4c0f606b43b050